website/www/site/data/en/quotes.yaml

```diff
@@ -0,0 +1,23 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# TODO:
+# Placeholder texts should be updated later
```

Comment:
We need to update placeholder texts for these quotes.

Reply:
done in https://github.com/apache/beam/pull/13596

Reply:
Great, thank you. I'll mark these resolved when that PR has been merged and the TODOs have been removed in website-revamp.